Update ko/tutorial/keras

rickiepark opened the pull request:
Updates for recent changes in en folder.
Running with Colab is Ok.
Thanks! :)

lamberta commented on site/ko/tutorials/keras/basic_classification.ipynb:
```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
I think you can ignore this for now. We don't like the way this looks and are working with the OSS team to make this better

lamberta commented on site/ko/tutorials/keras/basic_regression.ipynb:
```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Same. Remove from this PR, please

lamberta commented on site/ko/tutorials/keras/basic_text_classification.ipynb:
```diff
@@ -49,6 +49,8 @@
         "#\n",
         "# Copyright (c) 2017 François Chollet\n",
         "#\n",
+        "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Remove from this PR, please

lamberta commented on site/ko/tutorials/keras/overfit_and_underfit.ipynb:
```diff
@@ -59,6 +59,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Remove from this PR, please

lamberta commented on site/ko/tutorials/keras/save_and_restore_models.ipynb:
```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Remove from this PR, please

rickiepark commented:
Thanks for your comment, I remove the tags.

lamberta commented:
Hi Haesun,
You'll need to sync basic_regression for the changes to `plot_history` from https://github.com/tensorflow/docs/pull/305

Also, all notebook translations now require the `unicode_literals` module, just include it with the other imports like:

```
from __future__ import absolute_import, division, print_function, unicode_literals
```

THanks

rickiepark commented:
Hi Billy,
I added unicode_literals to notebooks in ko/tutorial/keras and updated plot_history function in basic_regression.ipynb.
Running with Colab is Ok.

I'm trying to translate notebooks in en/r2/tutorial/keras.
What do you think of it?

Thanks.

lamberta commented:
Thanks, Haesun!

As for r2/, yes, the r2/tutorials/keras notebooks should be stable since we're just migrating the r1 notebooks over.

But we'll need to make sure the latest changes to the r1/ notebooks get cherrypicked into r2/. I haven't been good at making sure people update both places :/

rickiepark commented:
Thanks @lamberta 
I'm trying to diff en/tutorial/keras to en/r2/tutorial/keras.
If necessary, I will make a PR to en/r2/tutorial/keras.
